### ISSUE TITLE
Date-/Time Picker - Use consistent option-attributes for button title/text

### ISSUE THREAD

**Issue by yves-s** [Dogfalo/materialize#5585]
### Consistent Date-Time Picker option-attributes for button title/text
```
{
          done: 'OK', // Text for done-button
          clear: 'Clear', // Text for clear-button
          cancel: 'Cancel' // Text for cancel-button
}
```

### Date Picker Options
```
{
          close: 'Ok',
          clear: 'Clear'
}
```

### Time Picker Options
```
{
          donetext: 'OK', // text for done-button
          cleartext: 'Clear', // text for clear-button
          canceltext: 'Cancel' // Text for cancel-button
}
```

## Context
It was my fault debugging the time picker and looking at the documented date picker options but I didn't think of such difference between the same elements. So I'd probably say it just makes sense.

**Comment by Dogfalo** [comment]
Good point, probably going to move the timepicker options into i18n as well.

**Comment by Dogfalo** [comment]
Fixed in f86c248f98e4e6e2526613f21af45e4bcf641826
